system-contract-dapp-playground/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center">
+<br />
+<a href="https://hedera.com/"><img src="./public/brandings/hedera-logo.png" width="150" alt=""/>
+<h1 align="center"> Hedera System Smart Contracts Dapp Playground </h1>
+<h5 align="center"> 🪜 Work In Progress... 🪜</h5>
+</p>
+
+## Overview
+
+TBD
+
+## Getting started
+
+### Requirements
+
+- [git](https://git-scm.com/)
+- [npm](https://www.npmjs.com/)
+- [nodejs](https://nodejs.org/en/)
+- [contributing guide](https://github.com/hashgraph/.github/blob/main/CONTRIBUTING.md#pull-requests)
+- [Hedera accounts](https://docs.hedera.com/hedera/getting-started/introduction#create-hedera-portal-profile-faucet)
+- [prettier pluggin](https://prettier.io/) (recommended)
+
+## Quickstart
```

Comment:
Add a `## Config` section and note all the different config files and their purposes.
There's a few more than usual so let's get them noted either here or in the next PR